### ISSUE TITLE
debian on SLS

### ISSUE THREAD

**Issue by deb75** [linux-surface/linux-surface#995]
Hello,

From the feature matrix, it seems that the SLS is fairly well supported,
although some features seems still experimental, is the pen already
usable ? is pressure handled ?

Also, did anyone successfully installed debian on it ?

Regards

**Comment by NP-chaonay** [comment]
1)is the pen already
= Yes
2) ... usable ? 
=yes
3)is pressure handled ?
=yes
4)Also, did anyone successfully installed debian on it ?
= me is Ubuntu 22.04 LTS (Ubuntu based on Debian)

**Comment by qzed** [comment]
Note that for touch and pen you'll need ITHC and IPTSd from the git repos.

**Comment by rforzachamp821** [comment]
Also, it seems as if i can't compile kernel modules for NVIDIA drivers on Debian (bookworm), as I get reports from dpkg that the surface kernel was compiled on Ubuntu (and i am compiling on debian). Setting environment variable IGNORE_CC_MISMATCH didn't work either.

**Comment by qzed** [comment]
Yes, until https://github.com/linux-surface/kernel/pull/132 is merged. I will update the Announcement issue once that is done.

**Comment by deb75** [comment]
Hi,

The pull request https://github.com/linux-surface/kernel/pull/133 is merged, do I still need to retrieve ITHC and IPTSD from their git repo ?

Regards

**Comment by qzed** [comment]
No, everything is included in the kernel and our IPTSd package now.
